src/helpers.js

```diff
@@ -89,6 +89,32 @@ module.exports = ({ types: t }) => {
     return scope.hasOwnBinding(name) ? scope.generateUidIdentifier(name) : t.identifier(name);
   };
 
+  const isFunctionExpression = factory => {
+    return t.isFunctionExpression(factory) || t.isArrowFunctionExpression(factory);
+  };
+
+  const createFunctionExpression = (factory, requireExpressions) => {
+    if (t.isFunctionExpression(factory)) {
+      return t.functionExpression(
+        null,
+        [],
+        t.blockStatement(requireExpressions.concat(factory.body.body))
+      );
+    }
+    let bodyStatement;
+    if (t.isBlockStatement(factory.body)) {
+      bodyStatement = factory.body.body;
+    } else {
+      // implicit return arrow function
+      bodyStatement = t.returnStatement(factory.body);
+    }
+    return t.arrowFunctionExpression(
+      [],
+      t.blockStatement(requireExpressions.concat(bodyStatement)),
+      false
```

Comment:
Let's omit this argument since [false is the default](https://github.com/babel/babel/tree/master/packages/babel-types#arrowfunctionexpression)